metric-anomaly-data-model/src/main/avro/MetricAnomalyNotificationEvent.avdl

```diff
@@ -0,0 +1,9 @@
+@namespace("org.hypertrace.alert.engine.metric.anomaly.datamodel")
+protocol MetricAnomalyNotificationEventProtocol {
+  record MetricAnomalyNotificationEvent {
+    string event_condition_id;
```

Comment:
nit: In follow up PR, lets add violation value if possible.

Reply:
https://github.com/hypertrace/hypertrace/issues/266